sdk/iot/hub/tests/cmocka/test_az_iot_hub_client.h

```diff
@@ -6,3 +6,4 @@ int test_iot_sas_token();
 int test_iot_hub_telemetry();
 int test_iot_hub_client();
 int test_az_iot_hub_client_twin();
+int test_iot_hub_methods();
```

Comment:
I think the Mac build is going to keep failing without a new line after this.

Reply:
@#$@%@ it! You might be right.

Reply:
Fixed.